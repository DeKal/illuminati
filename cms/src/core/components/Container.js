import { styled } from '@material-ui/core/styles'

const Container = styled('div')(({ theme }) => ({
  height: '100%',
  margin: `${theme.spaces.small}px 0`,
  padding: `${theme.spaces.medium}px`,
  border: `1px solid ${theme.colors.grey1}`,
  boxShadow: '0 1px 1px rgba(0,0,0,.04)',
  background: theme.colors.primaryBg,
  flexGrow: 1
}))

export default Container
