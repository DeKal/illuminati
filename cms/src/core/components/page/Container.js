import { styled } from '@material-ui/core/styles'

const Container = styled('div')(({ theme }) => ({
  background: theme.colors.background,
  padding: theme.spaces.medium,
  minHeight: theme.heightWithoutNavbar
}))

export default Container
