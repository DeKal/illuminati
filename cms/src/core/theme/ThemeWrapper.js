import React from 'react'
import { ThemeProvider } from '@material-ui/styles'
import PropTypes from 'prop-types'
import { theme } from './theme'

const ThemeWrapper = ({ children }) => {
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>
}

ThemeWrapper.propTypes = {
  children: PropTypes.any
}
export default ThemeWrapper
