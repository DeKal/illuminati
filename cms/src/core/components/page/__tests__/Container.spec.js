import * as React from 'react'
import { mount } from 'enzyme'
import Container from '../Container'
import ThemeWrapper from 'core/theme/ThemeWrapper'

describe('PageSwitch Container', () => {
  it('should render Container with no error', function() {
    const wrap = mount(
      <ThemeWrapper>
        <Container />
      </ThemeWrapper>
    )
    expect(wrap.find('Styled(div)')).toMatchSnapshot()
  })
})
