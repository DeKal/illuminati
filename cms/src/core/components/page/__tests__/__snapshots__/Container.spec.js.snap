// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSwitch Container should render Container with no error 1`] = `
<Styled(div)>
  <div
    className="div-root-1 div-root-2"
  />
</Styled(div)>
`;
